lib/portlayer/exec/handle.go

```diff
@@ -204,6 +204,17 @@ func (h *Handle) Commit(ctx context.Context, sess *session.Session, waitTime *in
 	return nil
 }
 
+// EmergencyCommit sets the handle's spec to nil so that Commit operation only does a state change and won't touch the extraconfig
+func (h *Handle) EmergencyCommit(ctx context.Context, sess *session.Session, waitTime *int32) error {
```

Comment:
We could, but then that line could be gone missing during a refactor etc.. I thought having it as a function will make more sense. 

Renamed it to CommitWithoutSpec, let me know if you have suggestions.